src/generator-filters.cs

```diff
@@ -353,10 +356,9 @@ void GenerateFilterSetter (string propertyType, string propertyName)
 			break;
 		case "string":
 			// NSString should not be added - it should be bound as a string
-			print ("using (var ns = new NSString (value))");
-			indent++;
-			print ("SetValue (\"{0}\", ns);", propertyName);
-			indent--;
+			print ("var ptr = CFString.CreateNative (value);");
+			print ($"SetHandle (\"{propertyName}\", ptr);");
```

Comment:
Would it be better to add a `SetValue` overload that takes a string, and does the CFString creation there? It would result in less generated/duplicated code.

Reply:
I like that. Let me have a look...